Source/Core/DolphinQt2/Config/Graphics/GraphicsSlider.h

```diff
@@ -0,0 +1,23 @@
+// Copyright 2017 Dolphin Emulator Project
```

Comment:
I think this comes down to preferences, but I'd be fine implementing it this way, although not sure that's really necessary.